doc/EFFECT_ON_CONDITION.md

```diff
@@ -624,12 +624,12 @@ checks do you have aikido stance active
 | ✔️ | ✔️ | ✔️ | ❌ | ❌ | ❌ |
 
 #### Examples
-
+You have equipped an item that you can stow
 ```json
 "u_can_stow_weapon"
 ```
 
-You have equipped an item with
+You have equipped an item that you can not stow
```

Comment:
Does this mean the item cannot be unwielded, like a cbm weapon or does it mean that you lack a sheath capable of holding the weapon?

Reply:
According to https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/condition.cpp#L1245, both